tiledb/sm/cpp_api/filter.h

```diff
@@ -127,7 +127,7 @@ class Filter {
    */
   template <
       typename T,
-      typename std::enable_if<std::is_arithmetic<T>::value>::type* = nullptr>
+      typename std::enable_if<!std::is_pointer_v<T>, int>::type = 0>
```

Comment:
```suggestion
      typename std::enable_if_t<!std::is_pointer_v<T>, int> = 0>
```

You missed that. Or didn't it compile?

Reply:
Just missed it.